l10n_fr_department/view/res_country_department.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-  Copyright 2013-2020 GRAP (http://www.grap.coop)
+  Copyright 2013-2022 GRAP (http://www.grap.coop)
```

Comment:
FTR you could save yourself from ever having to update these Copyright lines.
When reading the copyright comment, the dates are to be read as "starting.." (and legally it'll be valid for up to 70 years from that point)

eg: `Copyright 2013 GRAP (http://www.grap.coop)`

Means it's copyright is valid from 2013 to 2083.
There's no need at all to put an end date, that end up being just noisy diffs, and it's totally useless.


https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated